google/resource-snippets/pubsub-v1/test_alternatives.sh

```diff
@@ -28,12 +28,23 @@ terraform plan -var="deployment=d1" -var="project_id=${TF_PROJECT_ID}"
 terraform apply -auto-approve -var="deployment=d1" -var="project_id=${TF_PROJECT_ID}"
 popd
 
+# KCC Configuration
+gcloud config set project $KRM_PROJECT_ID
+cp -R alternatives/krm /tmp/krm_${KRM_PROJECT_ID}
+pushd /tmp/krm_${KRM_PROJECT_ID}
+kubectl apply -f pubsub.yaml
```

Comment:
You'd need a `kpt cfg set . PUBSUB my-pubsub` here for the test to pass, since DM and TF configs use that name (`my-pubsub-topic`) right?